content/en/blog/news/sbom-guide.md

```diff
@@ -0,0 +1,346 @@
+title: "Software Bill Of Materials guide"
+date: 2022-07-17
+draft: false
+description: >
+  Here we discuss the concepts of SBOM, how it can help in securing your software, and Generation formats and tools.
+categories: [blog]
+keywords: [Community, GSoC, 2022, supply chain security, BOM, SBOM]
+slug: "SBOM-Guide"
+aliases: []
+author: Osama Magdy
+---
+## Introduction
+
+Before going through Software BOMs, we need to set the ground for a rising concern in the software industry which is Software Supply Chain Security.
```

Comment:
Since this is the first time replace Software BOMs with Software Bill Of Materials (SBOMs)

Reply:
Done